src/login/Login.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import "./Login.css"
+
+class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      password: '',
+      error: ''
+    };
+  }
```

Comment:
Looks good I would like to know if the empty string or the null is better for the error default